slate/source/index.html.md

```diff
@@ -43,7 +43,6 @@ The detailed change log for the artifact is available [here](https://github.com/
 -   **JWT:** JSON Web Token
 -   **IPLoA:** Identity Proofing Level of Assurance
 -   **LoA:** Level of Assurance
--   **LoaAs:** Levels of Assurance
```

Comment:
LoAs is the preferred acronym https://www.secureidnews.com/news-item/level-of-assurance-changes-for-digital-identity-get-real-in-new-nist-800-63-draft/

Reply:
OK, so s/LoaAs/LoAs/? It mostly seemed weird to me to include the plural of an initialism that was already there.